src/app/views/rolodex_view.js

```diff
@@ -1,6 +1,185 @@
+
+    // Create a ContactView for each task
+    this.cardList = [];
+    this.model.forEach(function(contact) {
+      this.addContact(contact);
+    }, this);
+
+    // Keep track of our form input fields
+    this.input = {
+      name: this.$('.contact-form input[name="name"]'),
+      phone: this.$('.contact-form input[name="phone"]'),
+      email: this.$('.contact-form input[name="email"]')
+    },
+    //These let us know if the model has been updated or if a new model has been added
+
+    this.listenTo(this.model, "update", this.render);
+    this.listenTo(this.model, "add", this.addContact);
+    this.listenTo(this.model, "remove", this.removeContact);
+  },
+
+  render: function() {
+    console.log("rendering the rolodex")
+
+    this.listElement.empty();
+
+    // Loop through the data assigned to this view
+    this.cardList.forEach(function(card) {
+
+    card.render();
+
+      // Add that HTML to our contact list
+    $("#contact-cards").append(card.render().$el);
+
+    }, this);
+
+    return this; // enable chained calls
+  },
+
+  events : {
+    'click .btn-save': 'createContact',
+    'click .btn-cancel': 'clearInput',
+    'click .contact-card': 'showModal',
+    'click #edit' : 'prepEdit',
+    'click .btn-update' : 'updateContact'
+  },
+
+  prepEdit : function(event){
+    console.log("preparing to update");
+    $(".btn-save").hide();
+    $(".btn-update").show();
+    $("#add-or-edit").html("Update Contact")
+
+    //pre-populate the form with the details of that contact
+    $('.contact-form input[name="name"]').val($("#name").html()),
+    $('.contact-form input[name="phone"]').val($("#phone").html()),
+    $('.contact-form input[name="email"]').val($("#email").html())
+
+    //intake the form info.
+    this.input = {
+      name: this.$('.contact-form input[name="name"]'),
+      phone: this.$('.contact-form input[name="phone"]'),
+      email: this.$('.contact-form input[name="email"]')
+    }
+  },
+
+  clearInput : function(event) {
+    console.log("clearing form input");
+    this.input.name.val('');
+    this.input.phone.val('');
+    this.input.email.val('');
+  },
+
+  updateContact : function(event){
+    console.log("updating contact");
+
+    //get the model that goes with this info
+    for(var i = 0; i< this.cardList.length; i++){
+
+      if (this.cardList[i].model.get("name") == $("#name").html()){
+        console.log("found it!")
+
+         this.cardList[i].model.set("name", this.$('.contact-form input[name="name"]').val())
+         this.cardList[i].model.set("email", this.$('.contact-form input[name="email"]').val())
+         this.cardList[i].model.set("phone", this.$('.contact-form input[name="phone"]').val())
+         this.render();
+      }
+    }
+
+    $(".btn-save").show();
+    $(".btn-update").hide();
+    $("#add-or-edit").html("Add a New Contact")
+
+    this.clearInput()
+  },
+
+  showModal : function(event) {
+    console.log("showing modal")
+    $("#contact-details").show();
```

Comment:
Backbone has it's own jQuery object in each view, el.$, which you can use to find elements within that view.  It would be better practice to use that going forward.